db/migrate/20230117113947_create_greetings.rb

```diff
@@ -0,0 +1,9 @@
+class CreateGreetings < ActiveRecord::Migration[7.0]
+  def change
+    create_table :greetings do |t|
+      t.string :message
+
```

Comment:
- [ ] One of this project requirements is that you should create a table for storing 5 messages, so in this case, you can use a seed file to create a function that will create those messages.

![messages](https://user-images.githubusercontent.com/53124548/213676224-11d444ab-f03c-490d-b61d-26247b2c96d3.png)